input/pagecontent/reference.md

```diff
@@ -1467,53 +1467,33 @@ The contents of a value set are defined by a set of rules. There are four basic
 | A single code | <code>* <span class="optional">include</span> {Coding}</code> | `* $SCT#961000205106 "Wearing street clothes, no shoes"` |
 | All codes from another value set | <code>* <span class="optional">include</span> codes from valueset {ValueSet}</code> | `* include codes from valueset http://hl7.org/fhir/ValueSet/data-absent-reason` |
 | All codes from a code system | <code>* <span class="optional">include</span> codes from system {CodeSystem}</code> | `* include codes from system http://snomed.info/sct` |
+| Codes that lie in the _intersection_ of value set(s) and (optionally) a code system | <code>* <span class="optional">include</span> codes from <span class="optional">system {CodeSystem}</span> and valueset {ValueSet1}<span class="optional">and {ValueSet2}...</span></code> | `* include codes from valueset http://hl7.org/fhir/ValueSet/units-of-time and http://hl7.org/fhir/ValueSet/age-units` |
 | Filtered codes from a code system | <code>* <span class="optional">include</span> codes from system {CodeSystem} where {filter1} <span class="optional">and {filter2}...</span></code> | `* include codes from system $SCT where concept is-a #254837009` |
 {: .grid }
 
-These basic rules can be combined and extended. When using codes from a value set, the rule may refer to more than one value set. The rule may also refer to a code system and one or more value sets. When a rule refers to more than one value set, or has a code system and one or more value sets, the applicable codes are those present in _all_ listed value set(s) (and the system, if present).  To instead add all the codes in _any_ of the value set(s) or system (that is, the _union_ of the value set(s) and system), specify them in separate rules.
 
-**Examples:**
-
-* Include codes present in all listed value sets:
+> **Note 1:**  When a single include rule includes more than item (code system or value set), the applicable codes are those present in _all_ listed items.
+> **Note 2:**  To add codes from multiple code systems or value sets (i.e., the union not the intersection), specify them in separate `include` rules.
+> **Note 3:** When an `include` rule has both a system and more than one value set, the code system must be first or last. 
+> **Note 4:**  An `include` rule MUST not have more than one code system (the intersection of two code systems is the empty set)
+> **Note 5:** Filters are code system dependent. See [below](#filters) for further discussion.
```

Comment:
These notes end up squashed into one long block, but an extra line between each seems to get them as separate indented blocks.
```suggestion
> **Note 1:**  When a single include rule includes more than item (code system or value set), the applicable codes are those present in _all_ listed items.

> **Note 2:**  To add codes from multiple code systems or value sets (i.e., the union not the intersection), specify them in separate `include` rules.

> **Note 3:** When an `include` rule has both a system and more than one value set, the code system must be first or last.

> **Note 4:**  An `include` rule MUST not have more than one code system (the intersection of two code systems is the empty set).

> **Note 5:** Filters are code system dependent. See [below](#filters) for further discussion.
```